autogpt_platform/backend/backend/notifications/scoring_test.py

```
@@ -0,0 +1,57 @@
+
+from backend.notifications.scoring import ANOMALY_FACTOR, compute_score
+
+
+def _score(status, **over):
+    args = dict(
+        cost_cents=10.0,
+        node_error_count=0,
+        has_activity=True,
+        first_success=False,
+        cost_baseline=None,
+    )
```

Ruff (0.16.2)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)